.github/workflows/ci.yml

```diff
@@ -10,20 +10,43 @@ on:
   pull_request:  # for all PRs regardless of its base branch
 
 jobs:
-  build:
+  build-test:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
```

Comment:
Let's run tests only with Node.js 14.x (for simplifying the docker orchestration)